src/processors/eth-sepolia-local.ts

```diff
@@ -0,0 +1,24 @@
+import { lookupArchive } from "@subsquid/archive-registry";
+
+import { NETWORK } from "../consts";
+import { start } from "../processor";
+
+start({
+  dbOptions: { stateSchema: "eth_sepolia_processor" },
+  // I went through forked blocks here:
+  // https://sepolia.etherscan.io/blocks_forked?p=1 and I could only find one
+  // case where re-org depth exceeded 5 blocks: 22 blocks if re-org happens, I
+  // am not sure about this, because squid is supposed to handle automatically,
+  // but if error happens, all we would have to do is re-run / re-deploy the
+  // squid. So re-deploying once a year (for sepolia) is fine.
+  finalityConfirmation: 6,
+  network: NETWORK.ETH_SEPOLIA,
+  rentalFactoryAddress: "0x2c2BBA22aA19Ba34bC5BA65e6c35Ce54DA36A33D",
+  rentalManagerAddress: "0xea0b609F81B3D7699a970e670ec471Daf687E5c2",
+  source: {
+    // archive: lookupArchive("sepolia", { type: "EVM" }),
```

Comment:
**note to self: need to make this processor two-stage:**

(i) process from archive for the range: from contract deployed up to the forked block

(ii) immediately after the above processor start a new one: indexes from anvil for the ranges: from forked block and without to block

also note: consider whether forked block number should be used inclusively or exclusively in the above. If you use archive to index up to and including that block and anvil contains the same state, you aren't running into any issues because indexer would not re-index, since it saves into the db what was the last block that it saw. As such, this should not yield any issues:

```
// first processor: archive indexer; range:
startBlock: <when contract was deployed>,
endBlock: <forked block number>
```

```
// second processor: anvil indexer; range:
startBlock: <forked block number>
```

Reply:
I've took a micro stab at this yesterday but it's kind of sucky. Because the processor takes over the process I couldn't just `await start(...` and run the second `start()`. Unsure if subsquid has control flow like this exposed. Probably not. A potential solution here would be to `bash` it or see if we can update the `commands.json`, which appears to be invoking shell commands also.

Reply:
going to leave this unresolved (interested in the startBlock / endBlock discussion; in case it resurfaces in the future). mentioned this conversation in fork readme.